### ISSUE TITLE
Unexpected operator in bash

### ISSUE THREAD

**Issue by mabawsa** [kdave/btrfsmaintenance#24]
when I run the scripts directly in bash I get the following warning:

`
sudo sh /usr/share/btrfsmaintenance/btrfs-scrub.sh                                                                       
`
`
/usr/share/btrfsmaintenance/btrfs-scrub.sh: 14: [: /home:/var/www:/var/opt/gitlab/git-data: unexpected operator
`

The script then continues fine. Just wondered if my config was ok?

`
BTRFS_BALANCE_MOUNTPOINTS="/home":"/var/www":"/var/opt/gitlab/git-data"
`

**Comment by Djelibeybi** [comment]
Your config should be:

`BTRFS_BALANCE_MOUNTPOINTS="/home:/var/www:/var/opt/gitlab/git-data"`

**Comment by mabawsa** [comment]
I tried this already but with the same result:
```
 sudo sh /usr/share/btrfsmaintenance/btrfs-balance.sh
/usr/share/btrfsmaintenance/btrfs-balance.sh: 14: [: /home:/var/www:/var/opt/gitlab/git-data: unexpected operator
```

config
```
BTRFS_BALANCE_MOUNTPOINTS="/home:/var/www:/var/opt/gitlab/git-data"
```

It also complains, with these settings in the `BTRFS_SCRUB_MOUNTPOINTS` section:

```
sudo sh /usr/share/btrfsmaintenance/btrfs-scrub.sh
/usr/share/btrfsmaintenance/btrfs-scrub.sh: 14: [: /home:/var/www:/var/opt/gitlab/git-data: unexpected operator
```

**Comment by kdave** [comment]
This looks same as #23 and fixed by #25 .

**Comment by kdave** [comment]
Fixed in 0.3.1
